examples/pytorch/ogb/ogbn-mag/hetero_rgcn.py

```diff
@@ -17,18 +17,16 @@ def extract_embed(node_embed, input_nodes):
     for ntype, nid in input_nodes.items():
         nid = input_nodes[ntype]
         if ntype in node_embed.embeds:
-            emb[ntype] = node_embed.embeds.pop(ntype)(nid)
+            emb[ntype] = node_embed.embeds[ntype](nid)
     return emb
 
-def RelGraphEmbed(graph,embed_size,exclude):
-    node_num={}
+def relgraphembed(graph, embed_size):
+    node_num = {}
     for ntype in graph.ntypes:
-        if ntype in exclude:
+        if ntype == 'paper':
             continue
-        node_num[ntype] = graph.number_of_nodes(ntype)
-    embeds = HeteroEmbedding(node_num,embed_size)
-    for node_type in embeds.embeds.keys():
-        nn.init.xavier_uniform_(embeds.embeds[node_type].weight)  
+        node_num[ntype] = graph.num_nodes(ntype)
+    embeds = HeteroEmbedding(node_num, embed_size)
```

Comment:
I think you need to add `embeds.reset_parameters()`.

Reply:
`embeds.reset_parameters()` is in the fuction `main` ,Line452.

Reply:
Got it. Thanks.